test/utility.h

```diff
@@ -156,39 +159,39 @@ class TestVm : public testing::TestWithParam<std::string> {
 public:
   TestVm() {
     engine_ = GetParam();
-    vm_ = newVm();
+    vm_ = MakeVm(engine_);
   }
 
-  std::unique_ptr<proxy_wasm::WasmVm> newVm() {
+  static std::unique_ptr<proxy_wasm::WasmVm> MakeVm(const std::string &engine) {
```

Comment:
nit: rename to `makeVm`. This repo's code adheres to [Envoy coding style](https://github.com/envoyproxy/envoy/blob/main/STYLE.md), which calls for function names to use camel case starting with a lower case letter.

Reply:
Good to know, thanks